pages/index.tsx

```diff
@@ -20,10 +20,15 @@ import { HomepageTopNav } from "components/product-pages/common/Topnav";
 import { PageId } from "page-id";
 import { Suspense } from "react";
 import { ThirdwebNextPage } from "utils/types";
+import { PRODUCTS } from "components/product-pages/common/nav/data";
 
 const TRACKING_CATEGORY = "homepage";
 
 const HomePage: ThirdwebNextPage = () => {
+  const filterProducts = (section: string) => {
+    return PRODUCTS.filter((p) => p.section === section && !!p.dashboardLink);
+  };
```

Comment:
Maybe putting this in a helper function somewhere would be better ? Or just here is fine for now ?

Reply:
I don't think you want to use the same filter I'm using, since you want to link to the landing pages and not the dashboard inside pages